### DISCUSSION TITLE
Picker的placeholder是否应该默认为true？

### DISCUSSION THREAD

**Discussion by Jungzl** [3lang3/react-vant#556] [General]
此处默认为true是有什么考量吗，我用下来的感觉是可以少写一些样板代码恢复该处的值为undefined,不过从vant/antd-mobile过来的用户会感到困惑，怎么多出来了一个请选择的item。

**Comment by 3lang3** [comment]
> 此处默认为true是有什么考量吗，我用下来的感觉是可以少写一些样板代码恢复该处的值为undefined,不过从vant/antd-mobile过来的用户会感到困惑，怎么多出来了一个请选择的item。

目的就像你说的，恢复该处的值为undefined，值和视图层同步，不需要的话将placeholder设为false就行。
默认为true其实是我自己图方便了，项目核心维护就我一个，所以制定api的时候，讨论的空间几乎没有
